Data Layer: Add Nested Data Update Notes in store.push() Topics (#8666)
DevExpress/devextreme-documentation: concepts/70 Data Binding/5 Data Layer/3 Data Modification/30 Integration with Push Services.md

    name: "Chart SignalR"
}

[note]

To update nested data, specify `changes.data` as a nested object:

    store.push([{ type: "update", data: { Address: { City: "Bentonville" } }, key: key }]);

[/note]

> After being notified, the **DataSource** can reapply sorting, filtering, grouping, and other data processing settings if you set the [reshapeOnPush](/api-reference/30%20Data%20Layer/DataSource/1%20Configuration/reshapeOnPush.md '/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#reshapeOnPush') property to **true**. If pushes come too frequently, specify [pushAggregationTimeout](/api-reference/30%20Data%20Layer/DataSource/1%20Configuration/pushAggregationTimeout.md '/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#pushAggregationTimeout') to aggregate them.

Reviewer: ```suggestion
If you set [reshapeOnPush](/api-reference/30%20Data%20Layer/DataSource/1%20Configuration/reshapeOnPush.md '/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#reshapeOnPush') to **true**, the **DataSource** reapplies sorting, filtering, grouping, and other data processing settings after it receives a notification. If pushes occur too frequently, set [pushAggregationTimeout](/api-reference/30%20Data%20Layer/DataSource/1%20Configuration/pushAggregationTimeout.md '/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#pushAggregationTimeout') to aggregate them.
```
